src/aiortc/rtcrtpsender.py

```diff
@@ -288,21 +327,80 @@ def _send_keyframe(self) -> None:
         Request the next frame to be a keyframe.
         """
         self.__force_keyframe = True
-
+    
+    def ml_predict(self, frame, counter):
+        frame_array = frame.to_ndarray()
+        frame_array = np.resize(frame_array, (1024, 1024, 3))
+        kps, src_index = model.extract_keypoints(frame_array)
```

Comment:
This was a preliminary experiment to try some ideas about the encoder's behavior. I'm planning to formulate it to feedback the media.py instead of having the model at the sender.